config/application.rb

```diff
@@ -194,6 +194,9 @@ def try_config_for(*args)
     config.three_scale.plan_rules = ActiveSupport::OrderedOptions.new
     config.three_scale.plan_rules.merge!(try_config_for(:plan_rules) || {})
 
+    config.three_scale.max_allowed_days = ActiveSupport::OrderedOptions.new
```

Comment:
Rename this as well (`account_suspension`)

Reply:
yeah, of course, if I rename it, I rename it all to be the same 👍 otherwise it wouldn't make any sense.

Reply:
I would not create a separate config file for this feature of deleting tenants, but I'm OK with doing it to concentrate all time-related/TTL configs. Just like I would too for the email addresses (I don't like them in settings.yml either).

Reply:
> but I'm OK with doing it to concentrate all time-related/TTL configs

I don't mind doing this as long as the ones related to this feature are grouped together then 😝

Reply:
My idea was to separate settings from features.
We could use rolling updates but really to me that does not apply for something global for all tenants.


* Settings => Mandatory or defaults settings.
* Features => Removing the if ThreeScale.on_premises
* Rolling Updates => Features by tenants.

Wdyt @3scale/system ?

Reply:
Makes sense to merge features and rolling updates then.

Reply:
It is not really the sameimo @didierofrivia `rolling_updates` is features for providers (including master, yes, but for each account) and `features` is for the whole system.